data/test-cases/plugins/ms-teams-sync/UI-design-1.0/Disconnected_user_with_linked_channel.md

```diff
@@ -0,0 +1,42 @@
+---
+# (Required) Ensure all values are filled up
+name: "Updated RHS after disconnecting a user from their account on MM"
+status: Active
+priority: Normal
+folder: UI-design-1.0
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Click on the `Disconnect button` in the RHS of MS Teams sync plugin, when the channels are linked from MM to MS Teams .
```

Comment:
Do we really need to write `MS Teams sync plugin` here?

Reply:
if updating, fix everywhere

Reply:
discussed